src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb

```diff
@@ -1414,7 +1414,7 @@ Module Program
 #If Win32 Then
     Dim MaxLimit As Integer = 67000
 #ElseIf Mac Then
-    Dim MaxLimit as Integer = 5000                                     
+    Dim MaxLimit as Integer = 5000
```

Comment:
The diff shows a number of unnecessary whitespace changes.

Reply:
I didn't change this code, this is something that happened in the merge or passibily the save with CodeMaid.

Reply:
@paul1956 Can you disable any automatic formatting rules you have with CodeMaid (or just disable it outright)?   In general, the compiler does not like taking in changes that aren't necessary as part of the functional change being made.